WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersViewModel.kt

```diff
@@ -158,11 +158,13 @@ class BloggingRemindersViewModel @Inject constructor(
 
     private fun togglePromptSwitch() {
         _bloggingRemindersModel.value?.let { currentState ->
+            analyticsTracker.trackRemindersIncludePromptPressed(currentState.isPromptIncluded)
```

Comment:
I assume we want to track the state of the switch after it is toggled, eg. if we tap on OFF switch we want to track with isPromptIncluded set to true. In this case we need to call this method after we update the state.

Reply:
Good catch! I've fixed it, thanks!